core/sql/parser/sqlparser.y

```diff
@@ -16026,13 +16027,30 @@ exe_util_init_hbase : TOK_INITIALIZE TOK_TRAFODION
 
 		 DDLExpr * de = new(PARSERHEAP()) DDLExpr(TRUE, FALSE, TRUE, FALSE,
                                                           FALSE, FALSE,
-							  FALSE, FALSE, FALSE,
+							  FALSE, FALSE, FALSE, FALSE,
```

Comment:
In the beginning there were very limited variation to DDLExpr class so param approach was ok.
But now that there are many more options and variations to DDLExpr, it will be better to
set the bit flags instead of adding new params.

Maybe someone can clean this up at some point and change the callers to set bit flags 
when DDLExpr class is created in parser.